extra/hip_wrapper.py

```diff
@@ -200,21 +200,23 @@ def getStructTypeForArgs(*args):
         fields.append((f'field{idx}', ctypes.c_void_p))
     return __get_struct(types, fields)
 
-  def updateKernelNodeParams(npwrapper:kernelNodeParamsWrapper, *args, grid=(1,1,1), block=(1,1,1), updated_args=None):
-    _, struct, _ = npwrapper.context
-    if updated_args is not None:
-      for i in updated_args:
-        setattr(struct, f'field{i}', (args[i] if args[i].__class__ is int else args[i]._buf))
-    else:
-      for i,d in enumerate(args):
-        setattr(struct, f'field{i}', (d if d.__class__ is int else d._buf))
+  def setKernelNodeParam(npwrapper:kernelNodeParamsWrapper, arg, id):
+    setattr(npwrapper.context[1], f'field{id}', (arg if arg.__class__ is int else arg._buf))
+
+  def setKernelNodeLaunchDims(npwrapper:kernelNodeParamsWrapper, launch_dims):
```

Comment:
`def setKernelNodeLaunchDims(npwrapper:kernelNodeParamsWrapper, grid, block)` and `*` it when you call